claat/fetch_test.go

```diff
@@ -16,12 +16,17 @@ package main
 
 import (
 	"bytes"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"strings"
 	"testing"
 
+	"path/filepath"
+
+	"testing/quick"
```

Comment:
Could you remove empty lines from the standard pkg imports? That way they will all line up nicely.